src/Symfony/Component/Form/Tests/VersionAwareTest.php

```diff
@@ -11,12 +11,20 @@
 
 namespace Symfony\Component\Form\Tests;
 
+/**
+ * @deprecated since Symfony 7.2, use feature detection instead.
+ */
 trait VersionAwareTest
 {
     protected static int $supportedFeatureSetVersion = 404;
 
+    /**
+     * @deprecated since Symfony 7.2, use feature detection instead.
+     */
```

Comment:
Is it relevant to put this PHPdoc if it's already on the class?

Reply:
The method is often inherited from our own abstract test case classes. So people might be using the trait without knowing it. By flagging the method as deprecated, I want to make sure, static analysis tools and IDEs pickup the calls to this method properly.

Reply:
Got it, thanks!